population/areas.go

```diff
@@ -4,10 +4,22 @@ package population
 type Area struct {
 	ID       string `json:"id"`
 	Label    string `json:"label"`
-	AreaType string `json:"area-type"`
+	AreaType string `json:"area_type"`
 }
 
 // GetAreasResponse is the response object for GET /areas
 type GetAreasResponse struct {
 	Areas []Area `json:"areas"`
 }
+
+// Area is an area model with ID and Label
+type AreaTypes struct {
+	ID         string `json:"id"`
+	Label      string `json:"label"`
+	TotalCount string `json:"total-count"`
```

Comment:
you probable wanted to change to `total_count` here

Reply:
Also shouldn't AreaTypes be in area_type.go? Same for GetAreaTypesParentsResponse. Actually we should have a contract.go for that..